src/share/qtcreator/externaltools/lupdate.xml

```diff
@@ -31,10 +31,11 @@
     <displayname>Update Translations (lupdate)</displayname>
     <category>Linguist</category>
     <order>1</order>
-    <executable>
-        <path>%{CurrentProject:QT_INSTALL_BINS}/lupdate</path>
+    <executable output="showinpane" error="showinpane" modifiesdocument="no">
```

Comment:
Shouldn't modifiesdocument be yes? Isn't showinpane the default for output/error, i.e., can we omit it as in the original code?

Reply:
showinpane removed, modifiesdocument changed to yes.